app/uk/gov/hmrc/agentclientrelationships/connectors/UsersGroupsSearchConnector.scala

```diff
@@ -49,6 +63,17 @@ class UsersGroupsSearchConnector @Inject()(
     extends HttpAPIMonitor {
   override val kenshooRegistry: MetricRegistry = metrics.defaultRegistry
 
+  def getFirstGroupAdminUser(groupId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[UserDetails] = {
```

Comment:
shouldn,t this return `Future[Option[UserDetails]]` ?

Reply:
I think you're right :)

Reply:
I should also change the other method in this connector (which I didn't author), as it has the same problem - group may not exist.